src/main/scala/main/scala.scala

```diff
@@ -1,11 +1,16 @@
 package main
-import main.solutions.MazeSolver.{Point, Runner}
+
+import main.solutions.MazeSolver.{Solver, Point}
 
 object scala {
   def main(args: Array[String]): Unit = {
-    println("Welcome!")
+    val maze = Array(
```

Comment:
Array is almost never used in Scala. container like List, or to higher level Seq are more common and have more capabillities